amazonlinux/content.md

```diff
@@ -4,70 +4,63 @@ Amazon Linux is provided by Amazon Web Services (AWS). It is designed to provide
 
 The Amazon Linux container image contains a minimal set of packages. To install additional packages, [use `yum`](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/managing-software.html).
 
-AWS provides two versions of Amazon Linux: [Amazon Linux 2](https://aws.amazon.com/amazon-linux-2/) and [Amazon Linux AMI](https://aws.amazon.com/amazon-linux-ami/).
+AWS provides three versions of Amazon Linux:
 
-**Please note, that Amazon Linux 2023 is currently in the Preview mode and is not recommended for production workloads. Review the Amazon Linux 2023 [documentation](https://docs.aws.amazon.com/linux/al2023/ug/what-is-amazon-linux.html) to learn more.**
+-	[Amazon Linux 2023](https://aws.amazon.com/linux/amazon-linux-2023/) (recommended, latest)
+-	[Amazon Linux 2](https://aws.amazon.com/amazon-linux-2/)
+-	[Amazon Linux AMI](https://aws.amazon.com/amazon-linux-ami/)
 
-For information on security updates for Amazon Linux, please refer to [Amazon Linux 2 Security Advisories](https://alas.aws.amazon.com/alas2.html) and [Amazon Linux AMI Security Advisories](https://alas.aws.amazon.com/). Note that Docker Hub's vulnerability scanning for Amazon Linux is currently based on RPM versions, which does not reflect the state of backported patches for vulnerabilities.
+For information on security updates for Amazon Linux, please refer to:
 
-%%LOGO%%
+-	[Amazon Linux 2023 Security Advisories](https://alas.aws.amazon.com/alas2023.html)
+-	[Amazon Linux 2 Security Advisories](https://alas.aws.amazon.com/alas2.html)
+-	[Amazon Linux AMI Security Advisories](https://alas.aws.amazon.com/)
 
-## Where can I run Amazon Linux container images?
+Note that Docker Hub's vulnerability scanning for Amazon Linux is currently based on RPM versions, which does not reflect the state of backported patches for vulnerabilities.
```

Comment:
Can you provide some examples of this happening? The team responsible is very interested in improving the active results. :bow: :heart: Thanks!

Reply:
this is an old assertion that I kept as-is in this update.
I will check with the relevant team and get back.